mscore/resourceManager.cpp

```diff
@@ -1025,7 +1025,7 @@ void ResourceManager::downloadInstallPlugin()
       new_package.package_name = button->property("name").toString();
       analyzePluginPage("https://musescore.org" + page_url, new_package);
       if (new_package.source == UNKNOWN) {
-            button->setText("Failed Try again.");
+            button->setText("Unknown plugin source.");
```

Comment:
Needs to be translatable, so `button->setText(tr("Unknown plugin source."));`

Reply:
Thanks. Fixed in https://github.com/musescore/MuseScore/pull/4833/commits/1d26ad066f4fdc3c5897d2e6c467a6de02ce32f4.